MSSQL Integration - Update documentation with extra windows authentication details (#6325)
grafana/agent: docs/sources/static/configuration/integrations/mssql-config.md

    [- <queries> ... ]]
```

### Authentication

> By default, the `USERNAME` and `PASSWORD` used within the `connection_string` should correspond with a SQL Server username and password.

Reviewer: ```suggestion
By default, the `USERNAME` and `PASSWORD` used within the `connection_string` corresponds to an SQL Server username and password.
```

> If the Grafana Agent is running in the same Windows domain as the SQL Server, then the parameter `authenticator=winsspi` can be used within the `connection_string` without any additional credentials in order to authenticate.

Reviewer: ```suggestion
If {{< param "PRODUCT_ROOT_NAME" >}} is running in the same Windows domain as the SQL Server, then you can use the parameter `authenticator=winsspi` within the `connection_string` without any additional credentials to authenticate.
```

Author: @clayton-cornell PRODUCT_ROOT_NAME does not exist for static docs.

```conn
sqlserver://@<HOST>:<PORT>?authenticator=winsspi
```

> Otherwise if you would like to use Windows credentials to authenticate (instead of SQL Server credentials), you can use the parameter `authenticator=ntlm` within the `connection_string`. The `USERNAME` and `PASSWORD` would then be the Windows username and password (domain may need to be prefixed to the username).

Reviewer: ```suggestion
If you want to use Windows credentials to authenticate, instead of SQL Server credentials, you can use the parameter `authenticator=ntlm` within the `connection_string`.
The `USERNAME` and `PASSWORD` would be the Windows username and password
The Windows domain may need to be prefixed to the username.
```

    [- <queries> ... ]]
```

### Authentication
By default, the `USERNAME` and `PASSWORD` used within the `connection_string` argument corresponds to a SQL Server username and password.

> If {{< param "PRODUCT_ROOT_NAME" >}} is running in the same Windows domain as the SQL Server, then you can use the parameter `authenticator=winsspi` within the `connection_string` to authenticate without any additional credentials.

Reviewer: ```suggestion
If Grafana Agent is running in the same Windows domain as the SQL Server, then you can use the parameter `authenticator=winsspi` within the `connection_string` to authenticate without any additional credentials.
```

Reviewer: I missed that this was in Static. The Static docs don't use the product variable
